notebooks/build/covid-19-el-paso.ipynb

```diff
@@ -0,0 +1,6756 @@
+{
```

Comment:
<p>Is it just augmented with the Metaculus community prediction?</p><p> </p><p>Or does it also include the El Paso-specific historical data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
I think the former, right?